### PR TITLE
Update rubygem-nokogiri to 1.11.3

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Needs https://github.com/theforeman/foreman-packaging/pull/6582 and https://github.com/theforeman/foreman-packaging/pull/6581

**Comment by ehelms** [comment]
This version is used in unit and acceptance tests today via gem install. This would address -- https://github.com/advisories/GHSA-vr8q-g5c7-m54m

**Comment by ehelms** [comment]
Given we are at the end of stabilization week, this can wait to be merged after branching is complete is out and cherry pick it in for Foreman 2.5 RC2+

**Comment by ehelms** [comment]
#6581 has been merged. I'd like to get approval on this PR before merging https://github.com/theforeman/foreman-packaging/pull/6582 to avoid an outage in our pipelines.

File: packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec
Comment (ekohl) [inline packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec:108]: This is odd. It's listed as `%exclude` here and later without `%exclude`.

**Review by ekohl — APPROVED** [review]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
This PR caught https://github.com/theforeman/foreman-packaging/pull/6717 which took me by surprise.

**Comment by ehelms** [comment]
This now contains updates from my digging into how `racc` is available on EL8. The package is a bundled provide that is part of the `ruby-libs` package on EL8.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Repoclosure is broken because it wasn't published after build?

**Comment by ehelms** [comment]
> Repoclosure is broken because it wasn't published after build?

Correct.

**Pull request merged**
